web/_includes/issues.html

```diff
@@ -0,0 +1,11 @@
+<ul>
+  {% for issue in include.issues %}
+  <li>
+    <a href="{{ issue.url }}">{{ issue.title }}</a>
+    {% if issue.nb_comments > 1 %}
```

Comment:
Nothing shown when only 1 comment? Or maybe description counts as 1 comment?

Reply:
I was lazy to handle the plural case. 🙈  I fixed it in the next commit.